keras_cv/layers/preprocessing/jittered_resize.py

```diff
@@ -230,3 +292,11 @@ def get_config(self):
             }
         )
         return config
+
+    @classmethod
+    def from_config(cls, config):
+        if isinstance(config["scale_factor"], dict):
```

Comment:
is this needed for the test to pass?

Reply:
I found that it is not needed. I have removed it.